index.html

```diff
@@ -42,7 +42,7 @@
       <div>
         <ul>
           <li class="section-3-link">
-            <p>“</p><a href="https://github.com/mimblewimble/grin/blob/master/doc/intro.md"><span>Introduction to MimbleWimble</span></a><p>” →</p>
+                        <p>“</p><a href="https://docs.grin.mw/wiki/introduction/mimblewimble/ecc/"><span>Introduction to MimbleWimble</span></a><p>” →</p>
```

Comment:
Can you keep the old formatting though? (for consistency sake)

Reply:
Are you referring to the extra spaces here?

Reply:
Yes